crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -1106,6 +1121,12 @@ where
             claim_parser: claim_parser.clone(),
         };
 
+        lazy_static! {
+            static ref SHUTDOWN_SIGNAL: Arc<Semaphore> = Arc::new(Semaphore::new(0));
+        }
+        let trigger_shutdown: Arc<Box<dyn Fn() + Send + Sync>> =
```

Comment:
Does this really need to be passed around as an fn? I would be happy with module scoped access to this

Reply:
Thank you for the idea, I shall give it a try.